OpenTrans.net/Feature.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace OpenTrans.net
+{
+    public class Feature
+    {
+        public string FName { get; set; }
+        public string FValue { get; set; }
+        public string FUnit { get; set; }
```

Comment:
The standard mentions that dtUNIT units should be used. Does it make sense to turn this string field into an enum? I'd be happy to help out implementing this if this makes sense for you.

Reply:
FUnit data type is dtSTRING and specifications tells us: 
> Standard measuring units should be used if possible (refer also to Type dtUNIT).
so its not obligatory and i would implement something like a dtUNIT to string helper (either an enum or whatever suit more), but since it's a string this helper is just a nice to have imho

Reply:
yes, maybe you are right. On the other hand, the idea behind OrderTrans is to transfer data between independent parties. So it probably makes sense not to allow free text in too many instances.

So a solution might be:
https://github.com/stephanstapel/BMECat.net/blob/master/BMECat.net/QuantityCodes.cs

I'll implement this as soon as the pull request is finished!